docs/src/main/sphinx/release/release-360.md

```diff
@@ -0,0 +1,70 @@
+# Release 360 (XXX Jul 2021)
+
+## General
+
+* Improve support for correlated subqueries with GROUP BY or LIMIT and 
+  complex correlated filter conditions. ({issue}`8554`)
+* Disallow `DROP SCHEMA` if the schema is not empty. ({issue}`8660`)
+* Fix query failure when there is a constant predicate on some
+  column `col` (e.g `col=1`), followed by  `ORDER BY col` and `LIMIT`. ({issue}`8535`)
+* Fix `SHOW CREATE SCHEMA` failure. ({issue}`8598`)
+* Fix query failure when running `SHOW CREATE SCHEMA information_schema`. ({issue}`8600`)
+* Improve performance of `WHERE` clause evaluation. ({issue}`8624`)
+* Reduce coordinator network load. ({issue}`8460`)
```

Comment:
Can mention on what conditions?

Reply:
It's general really, there was a race condition when sending task update requests, but I think it's too detailed to add to RN